.local/bin/workman

```diff
@@ -58,10 +58,27 @@ if [ ! -f ./${WORKMAN_CONFIG_FILE} ]; then
     echo "BUILD_COMMAND=\"python2 x.py build -j8\""
     echo "# Command to run after creating a new working directory."
     echo "AFTER_COMMAND=\"rustup toolchain link \$(basename \$(pwd))-stage1 ./build/x86_64-unknown-linux-gnu/stage1\""
+    echo
+    echo "# Command to run in order to continue working in a given working directory"
+    echo "SESSION_REATTACH_COMMAND=\"tmux new -A -s \\\"\\\${PROJECT_NAME}/\\\${TASK}\\\"\""
+    echo "# Command to run in order to start working in a given working directory"
+    echo "SESSION_ATTACH_COMMAND=\"${SESSION_REATTACH_COMMAND}\"\""
+    echo "# Command to run in order to stop working"
+    echo "SESSION_KILL_COMMAND=\"tmux kill-session -t \\\"\\\${PROJECT_NAME}/\\\${TASK}\\\"\""
     exit 1
 fi
+
+_load_config() {
```

Comment:
I don't think we need to introduce a function to do this.

We could keep all the `source ${WORKMAN_CONFIG_FILE_PATH}` lines as they are and then just move your variable definitions (L73-L75) to above where we first source the config file (I think around L23 would be good).

If the user has defined variables with those names in their config file, then their values will be overwritten when we source the config file (only the first time, subsequent `source`s will override the user's value, but with things like `TASK` set - which we only know when running specific subcommands - so the value might be different, which is why we `source` multiple times).

If the user hasn't defined variables with those names then it won't override the variables and the definitions you include at the top of the file will be used.